common/github.com/IBM/vpc-go-sdk/makefile

```diff
@@ -0,0 +1,25 @@
+# Makefile
```

Comment:
`common/github.com/IBM/vpc-go-sdk/golang-apiref.json` Please remove this file. It is not required here. Thanks